dependencies.yaml

```diff
@@ -451,3 +457,16 @@ dependencies:
             packages:
               - cupy-cuda11x>=12.0.0
           - {matrix: null, packages: [cupy-cuda11x>=12.0.0]}
+  test_libcuspatial:
```

Comment:
The naming on these things is really bad. I'm not sure what would be better. `test_libcuspatial_includes`? Suggestions would be most welcome!

Reply:
I find dependencies.yaml extremely confusing on the best days!

Reply:
(Sorry for the late response)

I would read `test_libcuspatial` as "Test dependencies for libcuspatial."

Reply:
Elsewhere in RAPIDS, we use [`depends_on_cudf`](https://github.com/rapidsai/cudf/blob/54d49fcea4e7ad73df21f0dbfe99097c635b1023/dependencies.yaml#L699), specifically for packages that are shipped by this repository. That might have been a better approach here, in retrospect.